src/reducers/jobAppReducer.js

```diff
@@ -0,0 +1,85 @@
+import Axios from 'axios'
```

Comment:
What is the difference between `singleAppReducer` and `jobAppReducer`? Can it be just one reducer? Seems like overkill for a small app.